README.md

```diff
@@ -1,34 +1,40 @@
 # SVMKit
```

Comment:
The commit summary and description lines should be < ~78 characters.  Otherwise github (and other tools) do a terrible job at hard wrapping.

<img width="586" alt="Screenshot 2024-09-30 at 23 20 17" src="https://github.com/user-attachments/assets/a049b8d4-a267-455b-9151-3cc4f36b054d">